src/app/window-handler.ts

```diff
@@ -760,6 +786,43 @@ export class WindowHandler {
         delete this.windows[ key ];
     }
 
+    /**
+     * Registers keyboard shortcuts or devtools
+     */
+    private registerGlobalShortcuts(): void {
+        globalShortcut.register(isMac ? 'Cmd+Alt+I' : 'Ctrl+Shift+I', this.onRegisterDevtools);
+
+        app.on('browser-window-focus', () => {
+            globalShortcut.register(isMac ? 'Cmd+Alt+I' : 'Ctrl+Shift+I', this.onRegisterDevtools);
+        });
+
+        app.on('browser-window-blur', () => {
+            globalShortcut.unregister(isMac ? 'Cmd+Alt+I' : 'Ctrl+Shift+I');
+        });
+    }
+
+    /**
+     * Verifies and toggle devtool based on global config settings
+     * else displays a dialog
+     */
+    private onRegisterDevtools(): void {
+        const focusedWindow = BrowserWindow.getFocusedWindow();
+        const { devToolsEnabled } = config.getGlobalConfigFields([ 'devToolsEnabled' ]);
+        if (focusedWindow && windowExists(focusedWindow)) {
+            if (devToolsEnabled) {
```

Comment:
we can move this condition to the above if block

Reply:
@VishwasShashidhar we cannot because first, we need to validate if the window exists and then validate if devTools enabled

Reply:
let's avoid `if else` chain and use early return logic